code/modules/client/preferences/clothing.dm

```diff
@@ -152,10 +173,14 @@
 	main_feature_name = "Underwear"
 	category = PREFERENCE_CATEGORY_CLOTHING
 	should_generate_icons = TRUE
+	can_randomize = FALSE
 
 /datum/preference/choiced/underwear/init_possible_values()
 	return assoc_to_keys_features(SSaccessories.underwear_list)
 
+/datum/preference/choiced/underwear/create_default_value()
+	return "Nude"
```

Comment:
this should default to some form of underwear, the heart boxers would be the comedy option

Reply:
tru